src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -385,7 +385,7 @@ import io.sentry.Sentry
 Sentry.reportFullyDisplayed()
 ```
 
-Time to initial display is dependent on having an active transaction bound to the scope. Ideally, it should be used along with [Android's Activity Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation), which starts a transaction and binds it to the scope automatically.
+Time to full display is dependent on having an active transaction bound to the scope. Ideally, it should be used along with [Android's Activity Instrumentation](/platforms/android/performance/instrumentation/automatic-instrumentation/#androids-activity-instrumentation), which starts a transaction and binds it to the scope automatically.
```

Comment:
I believe this was wrong. @stefanosiano can you confirm this?

Reply:
both of them depends on an active transaction, but yeah, this one was a copy/paste from the one above 😅